email_app/app/safenet_comm.js

```diff
@@ -293,13 +294,14 @@ export const storeEmail = (app, email, to) => {
       .catch((err) => {throw MESSAGES.EMAIL_ID_NOT_FOUND})
       .then((service) => app.mutableData.fromSerial(service.buf))
       .then((inboxMd) => inboxMd.get(CONSTANTS.MD_KEY_EMAIL_ENC_PUBLIC_KEY)
-        .then((pk) => writeEmailContent(app, email, pk.buf.toString())
+        .then((pk) => writeEmailContent(app, email, pk)
```

Comment:
I would pass the string as before, i.e. `pk.buf.toString()`, so `writeEmailContent` doesn't need to assume that the value is the output of a `get` which contains the `buf`, it only needs a public key string as input, the caller should be responsible for making sure a string is provided as it's the one which knows where the key is being extracted from. In the future you may one to provide a public key which is not coming from a `get`.